ConsistencyManager/ConsistencyManager.swift

```diff
@@ -388,7 +388,7 @@ public class ConsistencyManager {
     */
     public func cleanMemory() {
         dispatchTask { _ in
-            NSNotificationCenter.defaultCenter().postNotificationName(ConsistencyManager.kCleanMemoryAsynchronousWorkStarted, object: self)
+            NotificationCenter.default.post(name: Notification.Name(rawValue: ConsistencyManager.kCleanMemoryAsynchronousWorkStarted), object: self)
```

Comment:
I think this seems more swifty to me? So convert the static var to `Notification.Name(...`